system/netdata-lsb.in

```diff
@@ -1,10 +1,18 @@
-#!/bin/bash
-# SPDX-License-Identifier: GPL-3.0-or-later
+#!/usr/bin/env bash
 #
+# Netdata LSB start script
+#
+# Copyright:
+#         SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Author:
+#         Costa Tsaousis  <costa@netdata.cloud>
+#         Pavlos Emm. Katsoulakis <paul@netdata.cloud>
+
 ### BEGIN INIT INFO
 # Provides:          netdata
-# Required-Start:    $local_fs $remote_fs $network $named $time apache2 httpd squid nginx mysql named opensips upsd hostapd postfix lm_sensors
-# Required-Stop:     $local_fs $remote_fs $network $named $time apache2 httpd squid nginx mysql named opensips upsd hostapd postfix lm_sensors
+# Required-Start:    $local_fs $remote_fs $network $named $time
```

Comment:
Do we need `$named` ?

Reply:
I want to have a call on these, I had a discussion with Costa on the systemd as well.

Reply:
I consider those three as core OS services, so it should be okay, no?

Reply:
It seems i am missing something.

First of all i see 5 there:
 - $local_fs
 - $remote_fs
 - $network
 - $named
 - $time

Back to `$named`. I guess it is `bind` (dns server), am i right? Doesn't look core to me

Reply:
and the most important thing, the way those 4 entities are defined, we don't fail to install if they are absent

Reply:
yea sorry, i was in multiple conversations and leaked half of the info.
so the core services i was talking about are fs-related and network and time related.
named indeed not of such high importance.

BTW This is what we depend on systemd too. systemd has a few extra (mysql, postfix) that we might consider removing after all, but that's a different change

I think we can stick with this PR for now and re-iterate once we conclude whether we plan to depend on any services or not.